power_grid_model_c/power_grid_model/include/power_grid_model/component/node.hpp

```diff
@@ -28,7 +28,7 @@ class Node final : public Base {
     explicit Node(NodeInput const& node_input) : Base{node_input}, u_rated_{node_input.u_rated} {}
 
     // update node, nothing happens here
-    static constexpr UpdateChange update(BaseUpdate const&) { return {false, false}; }
+    static constexpr UpdateChange update(BaseUpdate const& /*unused*/) { return {false, false}; }
```

Comment:
please use a more meaningful name.

```suggestion
    static constexpr UpdateChange update(BaseUpdate const& /*update_data*/) { return {false, false}; }
```

or even better: look at the update functions in the other components and use that same name